qiskit_nature/second_q/mappers/__init__.py

```diff
@@ -78,6 +78,16 @@
    :nosignatures:
 
    QubitConverter
+
+Tapered Qubit Mapper
```

Comment:
Can we move this *above* the `QubitConverter` section?

Also, it would be great if we can add one or two sentences to provide a little bit more context (similar to what is done for the `InterleavedQubitMapper` above. Just so that it is easier to see without navigating to the next page